files/scripts/bgp.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
+
+SERVICE="bgp"
+DEBUGLOG="/tmp/bgp_debug.log"
+
+function debug()
+{
+    /usr/bin/logger $1
+    /bin/echo `date` "- $1" >> ${DEBUGLOG}
+}
+
+function check_warm_boot()
+{
+    SYSTEM_WARM_START=`/usr/bin/redis-cli -n 6 hget "WARM_RESTART_ENABLE_TABLE|system" enable`
```

Comment:
use sonic-db-cli instead

Reply:
Changed  "/usr/bin/redis-cli -n 6" to "$SONIC_DB_CLI STATE_DB"